ansible/roles/installation/steamvr/tasks/main.yml

```diff
@@ -0,0 +1,90 @@
+---
+- name: Update package lists
+  apt: update_cache=yes
+  become: yes
+  
+- name: Intall needed libs for steamvr
+  apt: name={{ item }} install_recommends=no
```

Comment:
the same here
<pre>
apt:  
   name:   
   install_reco...  
</pre>

Reply:
done